misc/testerina/modules/testerina-core/src/main/ballerina/Module.md

```diff
@@ -174,7 +182,11 @@ _test.bal_
 The following example shows different ways of stubbing a function call.
 
 ```ballerina
-@test:Mock { functionName: "intAdd" }
+import ballerina/test;
```

Comment:
Do we need to explicitly include `import ballerina/test;` in the samples? its a bit inconsistent throughout the document. 

WDYT?

Reply:
Can these code snippets be considered examples that are meant to be copied to a project or just references?

Reply:
Let's add the imports wherever applicable.
These are considered to be examples IMO.

Reply:
Added imports where neccessary